traffic_portal/app/src/common/modules/table/types/table.types.tpl.html

```diff
@@ -34,18 +47,19 @@
             <thead>
```

Comment:
So somehow this table is getting an explicit size. I'm not sure if it can be overridden with SCSS, but I think ideally you want `width: 100%;` to fill up that `div.x_content` which I'm guessing is resized by angular/bootstrap. Currently (i.e. in master) this is never an issue because the explicit size appears to fit the available space, and is updated on window resize (to a point). However, if you remove all columns and thus the table does not display, then re-add some columns it seems to get stuck at a much smaller width (519px on my monitor, ymmv).

Idk if you can fix that, it might be obscured within jQuery datatables or inside of the column re-order library, but hopefully adding
```scss
.x_content {
    table {
        width: 100%; // !important ? hopefully not...
    }
}
```
would work.

Reply:
good catch. worked...but i had to use `!important`

Reply:
eww. But that's libraries for you.